src/picosvg/svg.py

```diff
@@ -76,6 +78,7 @@
 
 
 _ATTRIB_W_CUSTOM_INHERITANCE = frozenset({"clip-path", "transform"})
+_OPACITY_ATTRIB = frozenset({"opacity"})
```

Comment:
why is this a set and not a (single) string?

Reply:
Because at one point I thought I'd want to custom-handle fill-opacity and stroke-opacity. It could just be a const str now.